docs/azrepos-users-and-tokens.md

```diff
@@ -0,0 +1,210 @@
+# Azure Repos: Access Tokens and Accounts
+
+## Different Credential Types
+
+The Azure Repos host provider supports creating multiple types of credential:
+
+- Personal Access Token
+- Azure Access Token (experimental)
+
+### Personal Access Tokens
+
+Historically, the only option supported by the Azure Repos host provider was
+Azure DevOps Personal Access Tokens (PATs).
+
+These PATs are only used by Azure DevOps, and must be [managed through the Azure
+DevOps user settings page](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=preview-page).
+
+PATs have a limited lifetime and new tokens must be created once they expire. In
+Git Credential Manager, when a PAT expired (or was manually revoked) this
+resulted in a new authentication prompt.
+
+### Azure Access Tokens (experimental)
+
+Azure Access Tokens is the generic term for tokens issued by Azure Active
+Directory for either Work and School Accounts (AAD tokens) or Personal Accounts
+(Microsoft Account/MSA tokens).
+
+Azure DevOps supports Git authentication using Azure Access Tokens as well as
+PATs. Azure Access Tokens created by Git Credential Manager are scoped to Azure
+DevOps only.
+
+Unlike PATs, Azure Access Tokens get automatically refreshed and renewed as long
+as you are actively using them to perform Git operations.
+
+These tokens are also securely shared with other Microsoft developer tools
+including the Visual Studio IDE and Azure CLI. This means that as long as you're
+using Git or one of these tools with the same account, you'll never need to
+re-authenticate due to expired tokens!
+
+## User Accounts (experimental)
+
+In versions of Git Credential Manager that support Azure Access Tokens, the user
```

Comment:
I'm not sure `Azure Access Tokens` is correct - many Azure properties have token concepts.   Perhaps 'Microsoft identity platform access tokens'? That's from the official docs, but is a mouthful! https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens

Reply:
Yeah.. but maaaan is that a mouthful! @bgavrilMS do you have any recommendations here for a shorthand of "Microsoft identity platform access token"?